index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, intial-scale=1.0"/>
+    <title>First Portfolio</title>
+    <link href="./style.css" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+    <!-- START NAVBAR SECTION-->
+    <nav>
+        <div class="navbar">
+            <img src="./img/image_geometry_1.svg" alt="image1"/>
+            <button type="button" class="nav-btn">
+                <div class="lines line1"></div>
+                <div class="lines line2"></div>
+                <div class="lines line3"></div>
+            </button>
+        </div>
+    </nav>
+    <!-- END NAVBAR SECTION-->
+
+    <!-- START HEADLINE SECTION-->
+    <section class="headline">
+        <div class="content">
+            <h1 class="author">Charlie Giorgania</h1>
+            <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+            <button type="button" class="collab-btn">Start collaboration</button>
+        </div>
+        <div class="headlineimg">
+            <img src="./img/image_geometry_2.svg" class="mobileimg" alt="image2"/>
+        </div>
```

Comment:
Kindly, I recommend you to put all the decorative images in the background of your website, with the "background-image" property, it can be only 1 image and adapt it to the size of the device or put several background images and position them in their corresponding places.